assets/components/pages/about/about.twig

```diff
@@ -0,0 +1,91 @@
+{% extends "@templates/base/base.twig" %}
+
+{% block content %}
+
+<div class="container-full">
+  <div class="hero">
+    <div class="hero-content-container">
+      <h1 class="hero-title">About</h1>
+    </div>
+    <div class="hero-img">
+      <img src="images/gallery/gallery_01.jpg" alt="IMAGE DESCRIPTION HERE" class="img-fluid">
+    </div>
+  </div>
+</div>
+
+<div class="mt-5">
+  <p>
+    L'EPFL est la plus cosmopolite des universités techniques européennes. Elle accueille des étudiants, professeurs et collaborateurs de plus de 120 nationalités. A vocation à la fois suisse et internationale, l’EPFL est centrée sur trois missions : l’enseignement, la recherche et le transfert de technologies. L’EPFL collabore avec un important réseau de partenaires comprenant notamment les autres universités et hautes écoles, les écoles secondaires et gymnases, l’industrie et l’économie, les milieux politiques et le grand public, ceci dans le but d’avoir un véritable impact pour la société.
+  </p>
+</div>
+
+<div class="container-full bg-gray-100 py-5 mt-5">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4">
+        {% include '@molecules/links-group/links-group.twig' %}
+      </div>
+      <div class="col-md-4">
+        {% include '@content-types/basic-page/basic-page.twig' %}
+      </div>
+      <div class="col-md-4">
+        {% include '@molecules/links-group/links-group-teaser.twig' %}
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container">
+  {% include '@content-types/news/news-latest-two.twig' %}
+</div>
+
+<div class="container-full bg-gray-100 py-5 mt-5">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4">
+        {% include '@content-types/basic-page/basic-page.twig' %}
+      </div>
+      <div class="col-md-4">
+        {% include '@molecules/links-group/links-group.twig' %}
+      </div>
+      <div class="col-md-4">
+        {% include '@content-types/basic-page/basic-page.twig' %}
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container-full bg-gray-100 py-5 mt-5">
+  <div class="overflow-hidden">
+    <div class="container">
+      <h2 class="mb-4 mt-4">The next events</h2>
+      {% include '@organisms/card-slider/card-slider.twig' %}
+    </div>
+  </div>
+</div>
+
+<div class="container-full bg-gray-100 py-5 mt-5">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-6">
+        {% include '@content-types/basic-page/basic-page.twig' %}
+      </div>
+      <div class="col-md-6">
+        {% include '@content-types/basic-page/basic-page.twig' %}
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container mt-5">
+  <div class="row">
+    <div class="col-md-6">
+      Quote
```

Comment:
We will surely forget this one... Please use standard [`blockquote`](https://epfl-idevelop.github.io/elements/#/doc/design--typography.html) or some `@TODO` to keep this in a WIP states.

Reply:
Yes mentionned here d0564ed and #159